pages/common/pdfjoin.md

```diff
@@ -1,15 +1,20 @@
 # pdfjoin
 
-> PDF merging utility.
+> PDF merging utility based on pdfjam.
+> More information: <https://github.com/rrthomas/pdfjam-extras>.
 
-- Merge two PDFs:
+- Merge two PDFs into one with the default suffix "joined":
```

Comment:
Could you add an example customizing the suffix?  Or change this example to also show how to customize the suffix, and use `{{joined}}` as the value?

Reply:
That would be `--suffix {{string}}`

Reply:
This example should show minimal usage, so yeah I can add a separate one.